code/main.py

```diff
@@ -0,0 +1,281 @@
+import Featurizer
+from tkinter import *
```

Comment:
Instead of using the wildcard, I specified the namespace with 
```python  
import tkinter as tk
```
I'm not totally sure why, but the other imports statement seems necessary.
```python
from tkinter import ttk
from tkinter import filedialog
from tkinter import messagebox
```

The python manual says that I need something like this
```python
from tkinter import *
from tkinter.ttk import *
```
to overide the basic tk widgets with ttk, and the same goes for filedialog and messagebox.
```python
from tkinter import * 
from tkinter import messagebox 
```